packages/fxa-settings/package.json

```diff
@@ -42,7 +42,8 @@
     "react-dom": "^16.13.1",
     "react-scripts": "^3.4.1",
     "subscriptions-transport-ws": "^0.9.11",
-    "typescript": "3.8.3"
+    "typescript": "3.8.3",
+    "uuid": "^8.3.0"
```

Comment:
it'd be cool if we could either use `uuid@^7.0.3` here or update the other packages to 8. run `yarn why uuid`

Reply:
Oh neat. Thanks for catching this. Will use the existing version.